src/test/unit/algorithms/FlatSpatialPoolerTest.cpp

```diff
@@ -326,9 +326,11 @@ namespace nupic {
     NTA_CHECK(fsp_orig.getMinDistance() == fsp_dest.getMinDistance());
     NTA_CHECK(fsp_orig.getRandomSP() == fsp_dest.getRandomSP());
 
+#if !defined(NTA_OS_WINDOWS)
```

Comment:
Please remove. You can use `unlink` or `std::remove` rather than a `system` call.